docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+# "docker-compose up" to start the services
+version: "3"
```

Comment:
Any plans to extend this further for the `client` development?

Reply:
Yes. I have started working on it.